src/include/private/Makefile.in

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
what does this makefile do? does it do the copying to the "public" include directory?
(i.e., does the "private" directory show up in there?)
the "private" folder doesn't need to be copied there - it's not supposed to be accessible.